public/js/views/search-view.js

```diff
@@ -14,6 +14,26 @@ var app = app || {};
       return this;
     },
 
+    events: {
```

Comment:
Not sure where that came from, I definitely didn't write it. I may have pulled @smaia 's branch at some point